ext/standard/tests/strings/printf_error.phpt

```diff
@@ -25,16 +25,40 @@ $arg1 = 'one';
 $arg2 = 'two';
 
 echo "\n-- Call printf with one argument less than expected --\n";
-var_dump( printf($format1) );
-var_dump( printf($format2,$arg1) );
-var_dump( printf($format3,$arg1,$arg2) );
+try {
+    var_dump( printf($format1) );
+} catch (\Error $e) {
+    echo $e->getMessage(), "\n";
+}
+try {
+    var_dump( printf($format2,$arg1) );
+} catch (\Error $e) {
+    echo $e->getMessage(), "\n";
+}
+try {
+    var_dump( printf($format3,$arg1,$arg2) );
+} catch (\Error $e) {
+    echo $e->getMessage(), "\n";
+}
 
 echo "\n-- Call printf with two argument less than expected --\n";
-var_dump( printf($format2) );
-var_dump( printf($format3,$arg1) );
+try {
+    var_dump( printf($format2) );
+} catch (\Error $e) {
+    echo $e->getMessage(), "\n";
+}
+try {
+    var_dump( printf($format3,$arg1) );
+} catch (\Error $e) {
+    echo $e->getMessage(), "\n";
+}
 
 echo "\n-- Call printf with three argument less than expected --\n";
-var_dump( printf($format3) );
+try {
+    var_dump( printf($format3) );
+} catch (\Error $e) {
```

Comment:
Please catch ``\ValueError`` for all these try-catch blocks when applying the other changes.